src/siddhiCoreJava/LargestOfThree.java

```diff
@@ -0,0 +1,25 @@
+package siddhiCoreJava;
+
+public class LargestOfThree {
+
+	public static void main(String[] args) 
+	
+	{
+			int a = 13;
+			int b = 17;
+			int c =9;
+			
+			if (a>b && a>c)
+			{
+				System.out.println("a is greater");
```

Comment:
it should be like below code
System.out.println(a+ " is greater");

Reply:
Made the required changes